Jenkinsfile

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent {
+    label 'huge && bionic && x86_64 && !xenial && !trusty'
+  }
+
+  options {
+    timestamps()
```

Comment:
```

org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 9: Invalid option type "timestamps". Valid option types: [ansiColor, authorizationMatrix, buildDiscarder, catchError, checkoutToSubdirectory, copyArtifactPermission, disableConcurrentBuilds, disableResume, durabilityHint, lock, newContainerPerStage, overrideIndexTriggers, parallelsAlwaysFailFast, preserveStashes, quietPeriod, rateLimitBuilds, retry, script, skipDefaultCheckout, skipStagesAfterUnstable, timeout, waitUntil, warnError, withContext, withCredentials, withEnv, wrap, ws] @ line 9, column 5.
       timestamps()
       ^
```

Reply:
@djgalloway could you help install [timestamper](https://plugins.jenkins.io/timestamper/) to our jenkins?

Reply:
Thanks @tchaikov ! I commented out that line, pointing to that plugin.

Reply:
> @djgalloway could you help install [timestamper](https://plugins.jenkins.io/timestamper/) to our jenkins?

Done